File: term4/Kurs-projektowania-aplikacji-z-bazami-danych/project-old/src/main/kotlin/dbapp/project/domain/shipment/Shipment.kt
package dbapp.project.domain.shipment

import dbapp.project.domain.Entity
import dbapp.project.domain.shared.Address

data class Shipment(
        override val id: Int,
        val type: ShipmentType,
        val deliveryAddress: Address,
        var status: ShipmentStatus
) : Entity

File: term4/Kurs projektowania aplikacji z bazami danych/project/Store/Store.Domain/Model/Shipment/Shipment.cs
using Store.Common;
using Store.Domain.Model.Shared;

namespace Store.Domain.Model.Shipment
{
    public class Shipment : IEntity
    {
        public virtual int Id { get; set; }
        public virtual Address DeliveryAddress { get; set; }
        public virtual ShipmentType Type { get; set; }
        public virtual ShipmentStatus Status { get; set; }
    }
}
